packages/mds-web-sockets/clients.ts

```diff
@@ -2,12 +2,33 @@ import WebSocket from 'ws'
 import { WebSocketAuthorizer } from '@mds-core/mds-api-authorizer'
 import { AuthorizationError } from '@mds-core/mds-utils'
 import log from '@mds-core/mds-logger'
+import jwt, { GetPublicKeyOrSecret } from 'jsonwebtoken'
+import jwks from 'jwks-rsa'
+import { promisify } from 'util'
 
 export class Clients {
   authenticatedClients: WebSocket[]
 
   subList: { [key: string]: WebSocket[] }
 
+  private static getKey: GetPublicKeyOrSecret = async header => {
+    const client = jwks({
+      jwksUri: 'ourIssuer'
+    })
+
+    /* Technically, this typedef is slightly incorrect, but is to coerce the compiler to happiness without type guarding. One of publicKey or rsaPublicKey *always* exists. */
+    const key: { publicKey?: string; rsaPublicKey?: string } = await promisify(client.getSigningKey)(
+      header.kid ?? 'null'
+    )
+    return key.publicKey || key.rsaPublicKey
+  }
+
+  private static verify: (
+    token: string,
+    secretOrPublicKey: jwt.Secret | GetPublicKeyOrSecret,
```

Comment:
nit - maybe this arg should be named `publicKeyOrSecret` to match `GetPublicKeyOrSecret`?